### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by uonr** [argmaxinc/WhisperKit#277] [General]
I’ve tried many other Whisper tools, but most of them either lack essential features like stream transcription or VAD, or they aren’t optimized for Apple devices -- many simply don’t work on Apple devices. Then I discovered the sample application of WhisperKit, and it exceeded my expectations. It perfectly meets my requirements, even outperforming a paid app on the App Store that doesn’t support streaming. Honestly, I can’t believe this is just a “sample application,” aside from the basic UI.

I couldn’t find a donation link or anything, and I’m currently unable (have no money) to donate), but I just wanted to say thank you!

**Comment by atiorh** [comment]
Great to hear! Since you are eager to "donate", here are some ways to help:
- Build a project on top of WhisperKit and let us know the pain points
- Report issues with info to reproduce 
- Contribute features to the core Swift package

We collaborate with core contributors on our [Discord](https://discord.gg/G5F5GZGecC).
